### PR TITLE
Jira1234-adding new test

### PR DESCRIPTION
**Pull request by LaxmanRDec19**

### CODE REVIEW

File: src/test/java/PetApiTests.java [hunk 1]
```diff
@@ -39,5 +39,9 @@ public void testPostRequestAddPet() {
                 .body(pet).log().all().post().then().log().all().extract().response();
 
         System.out.println("status code: "+response.statusCode());
+
```
Comment (RahulVarma23) [inline src/test/java/PetApiTests.java]: remove all extra line

File: src/test/java/TestClass.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+import java.util.List;
+import java.util.Map;
+
+import static io.restassured.RestAssured.given;
+
+public class TestClass {
+
+    @Test
+    public void testPostRequestAddPet() {
+        String BASE_URI = "https://petstore.swagger.io/v2/pet";
```
Comment (RahulVarma23) [inline src/test/java/TestClass.java]: make it instance variable

File: src/test/java/TestClass.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+        Map<String, String > map = new HashMap<>();
+        map.put("Content-Type", "application/json");
+
+        Response response = given().headers(map).baseUri(BASE_URI)
+                .body(pet).log().all().post().then().log().all().extract().response();
+
+        System.out.println("status code: "+response.statusCode());
+        System.out.println("status code: "+response.statusCode());
+
```
Comment (RahulVarma23) [inline src/test/java/TestClass.java]: remove all extra lines

**Review by RahulVarma23 — CHANGES_REQUESTED** [review]

File: src/test/java/PetApiTests.java (hunk 1 above)
Comment (LaxmanRDec19) [reply src/test/java/PetApiTests.java]: done

File: src/test/java/TestClass.java (hunk 2 above)
Comment (LaxmanRDec19) [reply src/test/java/TestClass.java]: done

File: src/test/java/TestClass.java (hunk 3 above)
Comment (LaxmanRDec19) [reply src/test/java/TestClass.java]: done

**Review by RahulVarma23 — APPROVED** [review]

**Pull request merged**